src/material/datepicker/datepicker-input-base.ts

```diff
@@ -56,7 +56,9 @@ export class MatDatepickerInputEvent<D, S = unknown> {
   }
 }
 
-/** Function that can be used to filter out dates from a calendar. */
+/** Function that can be used to filter out dates from a calendar.
```

Comment:
The text looks good, but the formatting here is off. It should be on the next line and it needs an empty line before the `*/`.

Reply:
Thanks, updated 👍